code/drasil-data/Data/Drasil/SentenceStructures.hs

```diff
@@ -220,10 +221,12 @@ none :: Sentence
 none = S "--"
 
 found :: Double -> Sentence
-found x = (addPercent . realToFrac) (x*100)
-
+found x
```

Comment:
As discussed elsewhere, this hack is worse than the problem it supposedly fixes - I won't merge this in...